### PR TITLE
Support Django 1.11 and refactor travis/tests

### PR DESCRIPTION
**Pull request by orf**
I refactored the `.travis.yml` file to be a bit cleaner. I also added support for Django 1.11 (an import was broken), and fixed the flake8 violations that where in the project.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,12 +7,18 @@ env:
...
+  - TOX_ENV=py34-django111
   - TOX_ENV=py34-djangomaster
+  - TOX_ENV=py35-django18
+  - TOX_ENV=py35-django19
+  - TOX_ENV=py35-django110
+  - TOX_ENV=py35-django111
+  - TOX_ENV=py35-djangomaster
 install:
 - pip install tox
 script:
```
Comment (graingert) [inline .travis.yml]: can you change to `cache: pip` while you're here?

File: .travis.yml [hunk 2]
```diff
@@ -7,12 +7,18 @@ env:
   - TOX_ENV=py27-django18
```
Comment (graingert) [inline .travis.yml]: probably need to set `python: 3.5`

**Comment by orf** [comment]
Sorry, didn't see you leaving comments! I'm going to try and use tox-travis as well if you agree: https://pypi.python.org/pypi/tox-travis

Should simplify the config a bit.

**Comment by orf** [comment]
Hey @graingert, sorry for the commit spam. So, this MR:

1. Adds support for Django 1.11
2. Refactors the travis-ci.yml a bit
3. Fixes flake8 issues
4. Some style fixes in the template tags file

**Comment by orf** [comment]
I also got rid of the tests on django master, I don't think they are very valuable (and fail on 2.7). I think this PR halves travis build times as well.

Again, sorry for the spam, let me know if you have any comments or changes you want made.

**Comment by orf** [comment]
@akx - are you able to review this sometime soon? Lack of Django 1.11 support is really blocking us at the moment.

I can shift the individual fix out into a separate merge request if you like, and leave the test refactoring in this one? It's literally just an import that has been moved in Django 1.11. Nothing more.

**Comment by graingert** [comment]
closing in favour of pull #37

**Pull request closed without merging**